src/main/java/com/palantir/docker/compose/DockerComposition.java

```diff
@@ -108,4 +110,25 @@ public static DockerCompositionBuilder of(DockerCompose executable) {
         return new DockerCompositionBuilder(executable);
     }
 
+    public static DockerCompositionBuilder fromService(DockerService service) {
```

Comment:
I think we might just want to start using a builder except for a few exceptional easy cases. I'd suggest:
`DockerComposition.of(String... paths)` and `DockerComposition.of(DockerService... services)`.